### ISSUE TITLE
JavaFX applications fail to build with Maven 3.9.7

### ISSUE THREAD

**Issue by danielsz** [flow-storm/flow-storm-debugger#174]
Just a heads up about a serious problem with Maven 3.9.7. 

[Property replacement in dependency pom no longer works](https://issues.apache.org/jira/browse/MNG-8131)

JavaFx POM was invalid since 2021 and only worked "by chance". See [here](https://lists.apache.org/thread/4dqwyjhcy0ocp4jwlth3brj76wcr428g).

[Solution](https://mail.openjdk.org/pipermail/openjfx-dev/2024-May/047300.html) is to upgrade JavaFX to 23-ea+20.

**Comment by jpmonettas** [comment]
Hmm I was trying and openjfx 23 requires jdk >= 21, which will leave a bunch of FlowStorm users out. FlowStorm currently supports jdk >= 17.

**Comment by danielsz** [comment]
Yeah... It's a tough call... I'm in the same predicament as you... Meyvn supports FlowStorm and Morse, but I had to disable them because of the current issue. Temporarily, at least. JavaFX 22 was apparently compatible with JDK 17, but as you said, JavaFX 23 requires JDK21. Meyvn is compatible with JDK11, so I'm in no mood either to make it so restrictive.
